package.json

```diff
@@ -28,10 +28,10 @@
   "license": "Apache-2.0",
   "dependencies": {
     "express": "^4.16.4",
-    "firebase": "^6.2.4",
     "focus-visible": "^5.0.2",
     "lit-element": "^2.2.0",
     "navaid": "^1.0.5",
+    "terser": "^4.3.1",
```

Comment:
is this being used?

Reply:
Nope, I just installed it to compare minified sizes (we should use it later).

Reply:
FYI I removed this as part of the merge.